Arena.toml

```diff
@@ -1,358 +1,216 @@
-
-[[concerns]]
-kind = "LintWarning"
-document = "ENCODE-DCC/chip-seq-pipeline2:/dev/test/test_task/compare_md5sum.wdl"
-message = "[v1::W012::[Style, Clarity]::Low] string defined with single quotes (87:39-87:52)"
-
-[[concerns]]
-kind = "LintWarning"
-document = "ENCODE-DCC/chip-seq-pipeline2:/dev/test/test_task/compare_md5sum.wdl"
-message = "[v1::W012::[Style, Clarity]::Low] string defined with single quotes (91:18-91:27)"
-
-[[concerns]]
-kind = "LintWarning"
-document = "ENCODE-DCC/chip-seq-pipeline2:/dev/test/test_task/compare_md5sum.wdl"
-message = "[v1::W012::[Style, Clarity]::Low] string defined with single quotes (93:17-93:36)"
-
-[[concerns]]
-kind = "LintWarning"
-document = "getwilds/ww-fastq-to-cram:/ww-fastq-to-cram.wdl"
-message = "[v1::W001::[Spacing, Style]::Low] trailing space (133:46)"
+commit_hash = "a44b18a30fd6192af713fded4bc824f6e77f6ff9"
+filters = ["/template/task-templates.wdl"]
 
-[[concerns]]
-kind = "LintWarning"
+[[diagnostics]]
 document = "getwilds/ww-fastq-to-cram:/ww-fastq-to-cram.wdl"
-message = "[v1::W001::[Spacing, Style]::Low] trailing space (182:63)"
+message = "/ww-fastq-to-cram.wdl:129:5: note: task `FastqToUnmappedBam` has an extraneous parameter metadata key named `unmapped_bam` [rule: MatchingParameterMeta]"
 
-[[concerns]]
-kind = "LintWarning"
+[[diagnostics]]
 document = "getwilds/ww-fastq-to-cram:/ww-fastq-to-cram.wdl"
-message = "[v1::W001::[Spacing, Style]::Low] trailing space (34:105)"
+message = "/ww-fastq-to-cram.wdl:134:6: warning: task name `ValidateCram` is not snake_case [rule: SnakeCase]"
 
-[[concerns]]
-kind = "LintWarning"
+[[diagnostics]]
 document = "getwilds/ww-fastq-to-cram:/ww-fastq-to-cram.wdl"
-message = "[v1::W001::[Spacing, Style]::Low] trailing space (57:13)"
+message = "/ww-fastq-to-cram.wdl:165:5: note: task `ValidateCram` has an extraneous parameter metadata key named `validation` [rule: MatchingParameterMeta]"
 
-[[concerns]]
-kind = "LintWarning"
+[[diagnostics]]
 document = "getwilds/ww-fastq-to-cram:/ww-fastq-to-cram.wdl"
-message = "[v1::W001::[Spacing, Style]::Low] trailing space (8:42)"
+message = "/ww-fastq-to-cram.wdl:170:6: warning: task name `MergeBamsToCram` is not snake_case [rule: SnakeCase]"
 
-[[concerns]]
-kind = "LintWarning"
+[[diagnostics]]
 document = "getwilds/ww-fastq-to-cram:/ww-fastq-to-cram.wdl"
-message = "[v1::W003::[Completeness]::Medium] extraneous parameter meta within task: crai (203:5-203:9)"
+message = "/ww-fastq-to-cram.wdl:202:5: note: task `MergeBamsToCram` has an extraneous parameter metadata key named `cram` [rule: MatchingParameterMeta]"
 
-[[concerns]]
-kind = "LintWarning"
+[[diagnostics]]
 document = "getwilds/ww-fastq-to-cram:/ww-fastq-to-cram.wdl"
-message = "[v1::W003::[Completeness]::Medium] extraneous parameter meta within task: cram (202:5-202:9)"
+message = "/ww-fastq-to-cram.wdl:203:5: note: task `MergeBamsToCram` has an extraneous parameter metadata key named `crai` [rule: MatchingParameterMeta]"
```

Comment:
I'm going to remove the redundant relative path in `message` and just use the file name in the baselines.

Reply:
Nice. I was thinking about asking for that but wasn't sure how big an ask it would be.

Reply:
Of course, I comment on one where the file is at the root of the repo, so it's not terribly good example 🤦 

Many other files are not at the root, so repeating the path just makes it overly verbose.